server/bin/pbench-restore-and-unpack-tarballs.py

```diff
@@ -166,24 +165,36 @@ def main(options):
             cp_cmd = f"cp -a {tb} {tb}.md5 {ctrl_p}/"
             print(f"\n{cp_cmd}", flush=True)
             if not options.dry_run:
-                cp = subprocess.run(cp_cmd, shell=True, stderr=subprocess.STDOUT)
+                cp = subprocess.run(
+                    cp_cmd.replace('"', '\\"').replace("'", "\\'"),
```

Comment:
If you're going to keep the print at line 166, shouldn't the replacement at line 169 be done before the line is printed?  (You could probably append the `.replace()` invocations directly to the f-string in line 165.)

Reply:
Ditto line 181.